arviz/plots/autocorrplot.py

```diff
@@ -30,18 +31,24 @@ def plot_autocorr(
 
     Parameters
     ----------
-    data : obj
+    data: obj
         Any object that can be converted to an az.InferenceData object
         Refer to documentation of az.convert_to_dataset for details
-    var_names : list of variable names, optional
-        Variables to be plotted, if None all variable are plotted.
-        Vector-value stochastics are handled automatically.
-    max_lag : int, optional
+    var_names: list of variable names, optional
+        Variables to be plotted, if None all variable are plotted. Prefix the
+        variables by `~` when you want to exclude them from the plot. Vector-value
+        stochastics are handled automatically.
+    filter: Union[None, "like", "regex"], optional, default=None
```

Comment:
it should use docstring formatting, not type hints (type hints go in the function definition). In this case it would be:

    filter: {None, "like", "regex"}, optional

See references in https://github.com/arviz-devs/arviz/blob/master/CONTRIBUTING.md#docstring-formatting-and-type-hints

Reply:
My bad, will update, thanks 👌